### ISSUE TITLE
Module path resolve

### ISSUE THREAD

**Issue by timthurner** [Rockhopper-Technologies/pluginlib#5]
I've followed the directions perfectly for constructing the classes.

I have a directory structure like this  :

`
core/
┣ base_handlers/
┃ ┣ __init__.py
┃ ┣ base_processor.py
┃ ┣ base_scanner.py
┃ ┣ base_utilities.py
┃ ┗ file_config.json
┣ scanners/
┃ ┣ __init__.py
┃ ┗ scan_pdf.py
┣ __init__.py
┗ handler.py

`
Listed are source files accordingly :

` 
# core/scanners/base_handlers/base_scanner.py
import pluginlib

@pluginlib.Parent('scanner')
class Scannert(object):
    pass
` 

` 
# core/scanners/base_handlers/scan_pdf.py
import core.base_handlers.base_scanner as base_scanner

class PdfScan(base_scanner.Scanner):
    _alias_ = 'pdf'
` 

` 
# core/handler.py
   def load_plugins(self):
      import pluginlib
      import core.base_handlers.base_scanner
      self._pluginLoader = pluginlib.PluginLoader(paths=['scanners'])
      self.pluginLoader.plugins
`
It loads the base class fine, but it's dictionary is empty.

**Comment by avylove** [comment]
Yes, with your configuration I recommend using `PluginLoader(modules='core.scanners')`. Using `paths` is more for use cases where your modules are always installed in the same place on the system, such as in a subdirectory of `/usr/share`.
